### PR TITLE
avoid empty startedAt to crash console

### PR DESCRIPTION
**Pull request by ckiosidis**
# TL;DR
_Please replace this text with a description of what this PR accomplishes._

This code is failing if an execution is still on UNKNOWN phase.
When the execution transitions to QUEUED/RUNNING then it stops crashing

## Type
 - [X] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_
Screenshot 2021-09-29 at 15 30 46

### CODE REVIEW

**Review by jsonporter — APPROVED** [review]
lgtm

**Comment by ckiosidis** [comment]
@jsonporter could you merge this? I am not authorized to merge
